interfaces/builtin/mount_control.go

```diff
@@ -298,8 +299,22 @@ func enumerateMounts(plug interfaces.Attrer, fn func(mountInfo *MountInfo) error
 	}
 
 	for _, mount := range mounts {
+		types, err := parseStringList(mount, "type")
+		if err != nil {
+			return err
+		}
+
+		allowNoSource := false
+		if strutil.ListContains(types, "nfs") {
+			if len(types) > 1 {
```

Comment:
maybe relatedly to this we should have a comment here and maybe in the doc that supporting multiple types in one entry is mostly for situation when the filesystem is not provided but guessed or a single mount is expected to be performed with a few possible filesystems but under program control with other means